core/src/main/java/com/linecorp/armeria/internal/common/circuitbreaker/DefaultHttpCircuitBreakerClientHandler.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.common.circuitbreaker;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.client.circuitbreaker.CircuitBreaker;
+import com.linecorp.armeria.client.circuitbreaker.ClientCircuitBreakerGenerator;
+import com.linecorp.armeria.client.circuitbreaker.FailFastException;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.annotation.Nullable;
+
+import io.netty.util.AttributeKey;
+
+public final class DefaultHttpCircuitBreakerClientHandler
+        extends AbstractCircuitBreakerClientHandler<CircuitBreaker, HttpRequest, HttpResponse> {
+    private static final AttributeKey<CircuitBreaker> CIRCUIT_BREAKER =
```

Comment:
I realized this shouldn't be static but should be instantiated per handler instance (since multiple `DefaultHttpCircuitBreakerClientHandler` can be used with a single `ClientRequestContext`)
Let me fix this 🙏

Reply:
I missed this comment and just reviewed about it.